hr_recruitment_security/security/security.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <data>
+        <record id="group_hr_recruitment_officer_own_documents" model="res.groups">
+            <field name="name">Officer (own documents)</field>
+            <field name="category_id" ref="base.module_category_hr_recruitment"/>
+            <field name="implied_ids" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
+        </record>
+        <record model="ir.rule" id="hr_job_group_hr_recruitment_officer_own_documents_rule">
```

Comment:
You have to separate record rules in a `<data noupdate="1">` tag for allowing DB modifications (the same applies to my proposed rules).

Reply:
Yes, it's very important to add `noupdate`, thanks!